homeassistant/components/homeassistant/strings.json

```diff
@@ -1,20 +1,16 @@
 {
   "system_health": {
     "info": {
-      "installation_type": "Installation Type",
-      "version": "Version",
+      "arch": "CPU Architecture",
       "dev": "Development",
-      "virtualenv": "Virtual Environment",
-      "python_version": "Python Version",
       "docker": "Docker",
-      "arch": "CPU Architecture",
-      "timezone": "Timezone",
-      "os_name": "Operating System Name",
+      "installation_type": "Installation Type",
```

Comment:
You sort them alphabetically now. I had them sorted by how they appear on the UI :) (either works)